.pa11yci-pr.conf.js

```diff
@@ -96,7 +96,7 @@ var config = {
       url: '${HOST}/org/apikeys',
       wait: 500,
       rootElement: '.main-view',
-      threshold: 0,
+      threshold: 4,
```

Comment:
Had to increase the threshold as there is a false positive in alert component which is used here.